config.js

```diff
@@ -1441,6 +1441,23 @@ var config = {
     // dialInConfCodeUrl is the conference mapper converting a meeting id to a PIN used for dial-in
     // or the other way around (more info in resources/cloud-api.swagger)
 
+    // You can use external service for authentication that will redirect back passing a jwt token
+    // You can use tokenAuthUrl config to point to a URL of such service.
+    // The URL for the service supports few params which will be filled in by the code.
+    // tokenAuthUrl:
+    //      'https://myservice.com/auth/{room}?code_challenge_method=S256&code_challenge={code_challenge}&state={state}
+    // {room} - will be replaced with the room name
+    // {code_challenge} - (A web only). A oauth 2.0 code challenge that will be sent to the service. See:
+    // https://datatracker.ietf.org/doc/html/rfc7636. The code verifier will be saved in the sessionStorage
+    // under key: 'code_verifier'.
+    // {state} - A json with the current state before redirecting. This includes keys like room, roomSafe, tenant and
```

Comment:
Can we list exactly what we put in there? Also, wouldn't it make sense to put config overrides in a nested "config" key?

Reply:
List like all the whitelisted configs we have?

Reply:
Sorry I wasn't clear. I mean all the things we currently set, then say we support config overrides. The way you worded it made me believe there were more options we set, but reading the code I can see you listed them all.